content/millennium/r4/scheduling/appointment.md

```diff
@@ -27,6 +27,115 @@ When updating an appointment, the resource provides the ability to change the [A
+* [End date time](http://hl7.org/fhir/DSTU2/appointment-definitions.html#Appointment.end){:target="_blank"}
+* [Duration in minutes](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.minutesDuration){:target="_blank"}
+* [Comment](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.comment){:target="_blank"}
+* [Requested period](http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.requestedPeriod){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:appointment, :r4) %>
+
+## Search
+
+Search for Appointments that meet supplied query parameters:
+
+    GET /Appointment?:parameters
+
+_Implementation Notes_
+
+- Valid ids for the `practitioner` and `location` search parameters will be determined by the client and provided when
+  integrating your application with the client's production environment. See [overview](#overview) for details.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Parameters
+
+ Name          | Required?                                                | Type          | Description
+---------------|----------------------------------------------------------|-----------------------------------------------------------------------------------
+`_id`          | Yes, or one of `patient`, `practitioner`, or `location`. | [`token`]     | The logical resource id associated with the Appointment. Example: `3005759`
+`date`         | Yes when using `patient`, `practitioner`, or `location`. | [`date`]      | The Appointment date time with offset. Example: `2019-06-07T22:22:16.270Z`
+`patient`      | Yes, or `_id`                                            | [`reference`] | A single or comma separated list of Patient references. Example: `4704007`
+`practitioner` | Yes, or `_id`                                            | [`reference`] | A single or comma separated list of Practitioner references. Example: `2578010`
+`location`     | Yes, or `_id`                                            | [`reference`] | A single or comma separated list of Location references. Example: `633867`
+`status`       | No                                                       | [`token`]     | A single or comma separated list of appointment statuses. Example: `arrived`
+[`_count`]     | No                                                       | [`number`]    | The maximum number of results to return.
+
+Notes:   
+
+- The `patient`, `practitioner`, and `location` parameters may be included only once and may not be used in combination with the others.
+  For example, `patient=4704007,1316024` is supported but `patient=4704007&patient=1316024` and `patient=4704007&location=633867` are not.
+
+- The `date` parameter may be provided:  
+  - once with a prefix and time component to imply a date range. (e.g. `&date=ge2019-12-07T22:22:16.270Z`, `&date=lt2019-12-14T22:22:16.270Z`)     
```

Comment:
instead of 'imply a date range' this should be 'indicate a specific date/time' right?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/269/commits/6e094421299a9712e721c6d4a8d83ba8a4e36b87#diff-1d7df8ceb9f0ae8471323516e037c662R85